### DISCUSSION TITLE
Core features on users

### DISCUSSION THREAD

**Discussion by Guilhem185** [directus/directus#6132] [Archive]
I'm runnning v9.0.0.-rc69 and cannot see any revisions on users although "Track Activity & Revisions" is set on directus users system collection.
Same thing for exporting users, the response is "You don't have permission to access this.".
I guess 'directus users' is a specific collection, is there a way to use those functions on it or are they planned features?

**Comment by rijkvanzanten** [comment]
That sounds like a bug to me! Mind [opening an issue for this](https://github.com/directus/directus/issues/new/choose)?

**Comment by Guilhem185** [comment]
2 issues opened for these topics
https://github.com/directus/directus/issues/6243
https://github.com/directus/directus/issues/6244
Thanks
